lib/relation-definition.js

```diff
@@ -2540,8 +2539,8 @@ EmbedsMany.prototype.updateById = function (fkId, data, options, cb) {
     if (typeof cb === 'function') {
       modelInstance.updateAttribute(propertyName,
         embeddedList, options, function(err) {
```

Comment:
I am rather reluctant to change things that don't need to be changed, because unnecessary whitespace/formatting changes make code archaeology difficult. Additionally, I cannot fit all arguments on a single 80 char line :(